numpy/core/src/umath/loops.c.src

```diff
@@ -2780,20 +2784,20 @@ NPY_NO_EXPORT void
 @TYPE@_@kind@(char **args, npy_intp *dimensions, npy_intp *steps, void *NPY_UNUSED(func))
 {
     BINARY_LOOP {
-        const @ftype@ in1r = ((@ftype@ *)ip1)[0];
-        const @ftype@ in1i = ((@ftype@ *)ip1)[1];
+        @ftype@ in1r = ((@ftype@ *)ip1)[0];
```

Comment:
Hmm, now that I look at this, it is not quite the same as the sort ordering for complex numbers containing nans. Oh well.